### PR TITLE
Papers/deconvnet

### PR DESCRIPTION
**Pull request by czotti**

### CODE REVIEW

File: deep-learning/_posts/2017-03-31-deconv.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: "Learning Deconvolution Network for Semantic Segmentation"
+tags: CNN segmentation
+cite:
+  authors: "Hyeonwoo Noh, Seunghoon Hong, Bohyung Han"
+  title: "Learning Deconvolution Network for Semantic Segmentation"
+  venue: "2015, ICCV"
+pdf: http://www.cv-foundation.org/openaccess/content_iccv_2015/papers/Noh_Learning_Deconvolution_Network_ICCV_2015_paper.pdf
+---
+
+This papers present a neural network for semantic segmentation and classification. To achieve this they implement two new operation, the unpooling given a pooling map and the deconvolution.
```
Comment (Dref360) [inline deep-learning/_posts/2017-03-31-deconv.md]: presents, operations

File: deep-learning/_posts/2017-03-31-deconv.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+<div style="text-align:center">
+
+<img src="/deep-learning/images/deconv/maxpooling.png" />
+
+</div>
+
+### Deconvolution
+
+As shown in Figure 2 the deconvolution network is made of unpooling layer and deconvolution layer. The deconvolution is the operation to map one pixel into a higher dimensionnal space (see Figure 3), it's literally the opposite operation of the usual convolution. They crop the boundary of the enlarged activation of each deconvolution layer to keep the same size as the ouput of the previous unpooling layer.
```
Comment (Dref360) [inline deep-learning/_posts/2017-03-31-deconv.md]: layers

File: deep-learning/_posts/2017-03-31-deconv.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  title: "Learning Deconvolution Network for Semantic Segmentation"
+  venue: "2015, ICCV"
+pdf: http://www.cv-foundation.org/openaccess/content_iccv_2015/papers/Noh_Learning_Deconvolution_Network_ICCV_2015_paper.pdf
+---
+
+This papers present a neural network for semantic segmentation and classification. To achieve this they implement two new operation, the unpooling given a pooling map and the deconvolution.
+
+## Architecture
+
+It is a symetric VGG16 where on the left-end side it's the regular VGG16 and on the right-end side it's the same architecture with unpooling layer (see Figure 2) and deconvolution.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-31-deconv.md]: symmetric

File: deep-learning/_posts/2017-03-31-deconv.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+![](/deep-learning/images/deconv/architecture.png)
+
+
+### Unpooling
+
+When they process each maxpooling layer, they record the locations of maximum activations inside a pooled map for each maxpooling layers. They use these pooled map during the unpooling operation to retrieve the spatial information loose in the pooling operation (see Figure 3).
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-31-deconv.md]: * for each maxpooling layer
* these pooled maps
* loss

File: deep-learning/_posts/2017-03-31-deconv.md (hunk 2 above)
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-31-deconv.md]: * dimensional
* output

File: deep-learning/_posts/2017-03-31-deconv.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+</div>
+
+### Deconvolution
+
+As shown in Figure 2 the deconvolution network is made of unpooling layer and deconvolution layer. The deconvolution is the operation to map one pixel into a higher dimensionnal space (see Figure 3), it's literally the opposite operation of the usual convolution. They crop the boundary of the enlarged activation of each deconvolution layer to keep the same size as the ouput of the previous unpooling layer.
+
+## Results
+
+The result of their network are quite accurate and they try different post processing to enhance thei prediction.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-31-deconv.md]: * the results of their network are
* different post processing methods
* their prediction

File: deep-learning/_posts/2017-03-31-deconv.md (hunk 1 above)
Comment (ppoulin91) [inline deep-learning/_posts/2017-03-31-deconv.md]: paper

**Comment by jodoin** [comment]
please add the 'deep-learning' tag

File: deep-learning/_posts/2017-03-31-deconv.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+<div style="text-align:center">
+
+<img src="/deep-learning/images/deconv/maxpooling.png" />
+
+</div>
+
+### Deconvolution
+
+As shown in Figure 2 the deconvolution network is made of unpooling layer and deconvolution layer. The deconvolution is the operation to map one pixel into a higher dimensional space (see Figure 3), it's literally the opposite operation of the usual convolution. They crop the boundary of the enlarged activation of each deconvolution layer to keep the same size as the output of the previous unpooling layers.
```
Comment (jodoin) [inline deep-learning/_posts/2017-03-31-deconv.md]: layers.

It is literally

**Pull request merged**